controller/controller.js

```diff
@@ -0,0 +1,65 @@
+import data from '../models/model';
+
+class diary {
+  static getEntries(req, res) {
```

Comment:
Explain how this works

Reply:
The controller handle the logic behind the CRUD operation of user's requests before persisting data to the database(data structure in this case).  I created a class diary with static methods each handling user's requests.

-getEntries method handles the GET all entries request from the user. Once the request is made the method returns a response based on the entries stored.

Reply:
The controller handles the CRUD operations of user's request. It is made up of a class 'diary' with static methods handling each request of the user. I imported the model containing an array named 'data'. The exchange of data is done through the static methods in the controller class.

-getEntries method GET user's request and return response of all entries to the user stored within the data structure(array).

-getEntry method GET user's request, validates it and return response of an entry to the user stored within the data structure(array). If request is invalid, it returns a response with 'Not Found' message

-postEntry method handles POST user request, validates it and add an entry to the database with a response sent to the user if valid. If user request not valid, return with a response to user of 'entry not added'.

-modifyEntry method handles PUT user request validates it and return response of the entry modified  within the data structure(array). If request is invalid, it returns a response with 'Not Found' message

-deleteEntry method handles DELETE user request validates it and return response with message of entry deleted from the data structure(array). If request is invalid, it returns a response with 'Not Found' message